### ISSUE TITLE
make icon different than the minetest one

### ISSUE THREAD

**Issue by AFCMS** [dragonfireclient/dragonfireclient#33]
## Problem
The current icon of the dragonfire client is similar to minetest one.
You can confuse minetest and dragonfire in your taskbar.

## Solutions

Create a different icon

**Comment by LizzyFleckenstein03** [comment]
I tried and I failed.

**Comment by AFCMS** [comment]
> I tried and I failed.

Did you fail to design the icon?

**Comment by LizzyFleckenstein03** [comment]
No. I failed to make it so the icon is used. I would have to change the internal project name from "minetest" to dragonfire.

**Comment by N-nec** [comment]
> No. I failed to make it so the icon is used. I would have to change the internal project name from "minetest" to dragonfire.

Is this Minetest? or a Hacked client for players to bypass the Privs?

**Comment by fisherbait** [comment]
> Is this Minetest? or a Hacked client for players to bypass the Privs?

This is a hacked client that offers different features than the main Minetest client.

**Comment by AmyMoriyama** [comment]
Where there is a will there is a way.
